django/db/models/expressions.py

```diff
@@ -457,13 +457,17 @@ def desc(self):
         return OrderBy(self, descending=True)
 
 
-class Func(Expression):
+class Func(RegisterLookupMixin, Expression):
     """
     A SQL function call.
+
+    Note that `RegisterLookupMixin` is first so that get_lookup
```

Comment:
I like to include () after functions & method names to help distinguish them from attributes.